x-pack/examples/ui_actions_enhanced_examples/public/drilldowns/app2_to_dashboard_drilldown/app2_to_dashboard_drilldown.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  DashboardEnhancedAbstractDashboardDrilldown as AbstractDashboardDrilldown,
+  DashboardEnhancedAbstractDashboardDrilldownConfig as Config,
+} from '../../../../../plugins/dashboard_enhanced/public';
+import { SAMPLE_APP2_CLICK_TRIGGER, SampleApp2ClickContext } from '../../triggers';
+import { KibanaURL } from '../../../../../../src/plugins/share/public';
+
+export const APP2_TO_DASHBOARD_DRILLDOWN = 'APP2_TO_DASHBOARD_DRILLDOWN';
+
+type Trigger = typeof SAMPLE_APP2_CLICK_TRIGGER;
+type Context = SampleApp2ClickContext;
+
+export class App2ToDashboardDrilldown extends AbstractDashboardDrilldown<Trigger> {
+  public readonly id = APP2_TO_DASHBOARD_DRILLDOWN;
+
+  public readonly supportedTriggers = () => [SAMPLE_APP2_CLICK_TRIGGER] as Trigger[];
+
```

Comment:
Maybe let's overrides display name in these two. Instead of `Go to dashboard` make `Go to dashboard (app2)`

Reply:
No, the whole idea of the dashboard drilldown now is that it can be used in any app, I don't see why we would want to specify `(app2)`.